### DISCUSSION TITLE
HidHide Drivers v1.1.50

### DISCUSSION THREAD

**Discussion by nefarius** [nefarius/HidHide#44] [Announcements]
# v1.1.50 HidHide changelog

## How to install

Make sure to have the [prerequisites](https://support.microsoft.com/en-us/topic/the-latest-supported-visual-c-downloads-2647da03-1eea-4433-9aff-95f26a218cc0) installed. Download the provided setup and click through it. Reboot afterwards. Done!

The Symbol files are **optional** and only useful when debugging issues.

## Remarks

This release supports **Windows 10/11 64-Bit Intel/AMD only**.

## Features

- Added command line client for use in scripts and automation (#7)
- Client UI auto-whitelists itself upon launch to avoid confusion once blocking is engaged (#11, #16)

## Bugfixes

- Fixed building incorrect application path when NTFS junctions are used (#9)
- Improved driver performance by removing redundant I/O queue (#12)

<hr /><em>This discussion was created from the release <a href='https://github.com/ViGEm/HidHide/releases/tag/v1.1.50.0'>HidHide Drivers v1.1.50</a>.</em>

**Comment by SonyLie** [comment]
No work in windows 8.1 need 8.1 version windows, cose on windows 10 dont work videodrivers

**Reply by Kanuan** [reply]
![image](https://user-images.githubusercontent.com/24910442/168488657-789d8994-5fca-4dc5-a1c4-6643aaecc33e.png)

**Reply by SonyLie** [reply]
need 8.1 version windows, **cose on windows 10 dont work videodrivers**

**Reply by nefarius** [reply]
**There is no 8.x version** so you can stop asking for it. Upgrade your OS or leave it be or build it yourself, repeating your needs will not magically make things happen. Entitled people these days...

**Reply by SonyLie** [reply]
I dont have windows 10 coz i dond have video drivers for windows 10 no video drivers no game no game no need gamepad? All hide hide drivers another program work with windows 8, just create hide drivers for windows 8 or regret

**Reply by nefarius** [reply]
"or regret" lol get out of here, kiddo. You're embarrassing yourself. Make your own HidHide or shut up, as simple as that. You won't get Windows 8 support from us, period. Now kindly stop posting your pre-pubescent nonsense here and grow up.

**Reply by SonyLie** [reply]
I work in goverment russian federation boy

**Reply by nefarius** [reply]
Good for you. I'm even less scared now, Orc boy 😎

**Reply by SonyLie** [reply]
do program for win 8 or regret

**Reply by nefarius** [reply]
Go royally f yourself, child. Blocked.

**Reply by ImaDoor2785** [reply]
> I work in goverment russian federation boy

stupidest response ive ever heard XD

**Comment by CarefulCrab** [comment]
Hi I'm having a bit of a problem with the program. I completely followed the install instructions but when it comes to the step: "Open the "HidHide Configuration Client" in the start menu", it just doesn't show up on the start menu. It's as if the program isn't anywhere on my computer even though I just installed it following every single one of the recommended steps before this point.
Could you please help me out on how to fix this issue? Thank you.

**Reply by nefarius** [reply]
Guys, kindly open your own discussion instead of appending to the release post...
